### DISCUSSION TITLE
🎧 Kaizen 9 - Embracing change 🌟 - Mar. 2, 2023

### DISCUSSION THREAD

**Discussion by gerhard** [thechangelog/changelog.com#440] [Kaizen]
[![shields.io](https://img.shields.io/badge/Last%20updated%20on%20Mar.%202%2C%202023-PUBLISHED-blue?style=for-the-badge)](https://shipit.show/90)

[This is now available as shipit.show/90!](https://changelog.com/shipit/90) See all improvements in this 🐙 GitHub repository: [Kaizen 8 ... 9](https://github.com/thechangelog/changelog.com/compare/kaizen.8...kaizen.9)

<a href="https://changelog.com/shipit/90"><img width="1440" alt="Screenshot 2023-03-07 at 07 25 28" src="https://user-images.githubusercontent.com/3342/223354192-ca00a0ba-0791-410d-a16c-78ac531f936a.png"></a>

---

This contains all things related to the next Kaizen episode - https://shipit.show/90 🎧 Mar. 2, 2023

Share your thoughts via comments below:
- Is there something in changelog.com that you, our dear listener, would like us to tackle?
- Is there something that **you** would like to help out with?
- Anything that we - @adamstac @jerodsanto - want to improve?

This is the timeline that we are working with (red line is today):

```mermaid
gantt
    title 🎧 Kaizen 9 timeline - https://shipit.show/90
    dateFormat  YYYY-MM-DD
    axisFormat %b %e, %Y

    section DISCUSS
    ✍️ Add your comments below :2023-01-03, 48d

    section CONTRIBUTE
    💻 ALL CONTRIBUTIONS WELCOME 👕 :2023-01-03, 55d

    section RECORD
    🎙 Record :2023-02-27, 1d
    ✂️ Edit :2023-02-28, 2d
    🚀 SHIP IT! :milestone, 2023-03-02

    section NEXT
    Kick-off Kaizen 10 :2023-03-02, 7d
```

---

### 🎙 Proposed topics

- [x] Embracing change
- [x] Community improvements
  - 👕 Chris Eggert: [dev container spec](https://github.com/thechangelog/changelog.com/pull/437) + How to **Squash & merge** PRs?
  - [Jarvis Yang](https://changelog.slack.com/archives/C03SA8VE2/p1675971225382269): running locally + tooltips to show keyboard shortcuts on the onsite Player
<img width="985" alt="image" src="https://user-images.githubusercontent.com/3342/219851533-23d379fb-5b48-41a6-98f6-20cbe6e37229.png">

- [x] Our improvements
  - PRs now run tests [PR#436](https://github.com/thechangelog/changelog.com/pull/436)
  - Building & publishing our runtime image in CI [PR#450](https://github.com/thechangelog/changelog.com/pull/450) 👉 [ghcr.io/thechangelog/changelog-runtime](https://ghcr.io/thechangelog/changelog-runtime) - who would like to bump Elixir, Erlang & Node.js versions?
  - Writing all our CI/CD as Go code [PR#446](https://github.com/thechangelog/changelog.com/pull/446)
  - [Rotate all the secrets](https://github.com/thechangelog/changelog.com/issues/442) + 1Password Service Accounts (Jan. 15) + Stats disruption AWS S3
- [x] What comes next?
  - [Sentry errors](https://changelog-media.sentry.io/issues/?project=5668962&query=is%3Aunresolved&referrer=issue-list&sort=freq&statsPeriod=30d)
  - 1Password Connect Server?
  - [Audit trail of all production changes](https://github.com/gerhard/changelog.com/tree/track-commits-into-production) with Kosli.com
  - Kaizen 10: where & when?

<img width="1091" alt="image" src="https://user-images.githubusercontent.com/3342/219851358-cc52f101-b4e3-496c-85f4-438fac962893.png">

---

### What changed so far? [Kaizen.8 ... Mar. 2](https://github.com/thechangelog/changelog.com/compare/kaizen.8...27dde37e11466f9d5a03b1a6782a584a0fc6935c)
- ✅ PRs now run tests - bundled in https://github.com/thechangelog/changelog.com/pull/436
- ✅ https://github.com/thechangelog/changelog.com/pull/437 - 🙏 Thank you @pilor 
  - https://github.com/thechangelog/changelog.com/pull/449
- ✅ https://github.com/thechangelog/changelog.com/pull/443 - 😢 @akoutmos
- ✅ https://github.com/thechangelog/changelog.com/issues/442
- ✅ https://github.com/thechangelog/changelog.com/pull/446
- ✅ https://github.com/thechangelog/changelog.com/pull/450

### What else is coming up?
- track commits into production - https://github.com/gerhard/changelog.com/tree/track-commits-into-production
- track runtime changes

---

This is a follow-up to:
- https://github.com/thechangelog/changelog.com/discussions/433

**Comment by jerodsanto** [comment]
We now have a [development container](https://containers.dev) config with instructions for VS Code's web editor thanks to PR #437. This is definitely worth a mention on Kaizen 9.

I'm also considering an episode of The Changelog all about dev containers. Wonder who the best guest would be for that...

**Comment by adamstac** [comment]
I agree. We could ping @filmgirl and see who she might suggest. Should they be non-GitHub?

**Reply by jerodsanto** [reply]
I reached out to @bamurtaugh on LinkedIn. She's a [primary contributor to the spec's repo](https://github.com/devcontainers/spec/graphs/contributors). Seems like a good place to start!

**Reply by bamurtaugh** [reply]
Hey there! I'd love to connect to learn more.

My email is brmurtau@microsoft.com, if you'd like to start an email thread or set up some time to chat. Thanks so much for your interest in dev containers and for reaching out!
